workspaces/icecap-host/Makefile

```diff
@@ -65,10 +65,12 @@ build-veracruz-server-test: rustup-plat $(VERACRUZ_ICECAP_QEMU_IMAGE)
 		) target/release/veracruz-server-test
 
 .PHONY: $(MEASUREMENT_FILE)
```

Comment:
`$(MEASUREMENT_FILE)` is a target in `shared.mk`, but if it's going to be phony then it might as well be a simple name rather than a variable expression. I'll try changing it to `measurement-file`.